dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/gdv/GdvUmweltRollingWindowComponents.kt

```diff
@@ -0,0 +1,278 @@
+package org.dataland.frameworktoolbox.frameworks.gdv
+
+import org.dataland.frameworktoolbox.frameworks.gdv.custom.GdvYearlyDecimalTimeseriesDataComponent
+import org.dataland.frameworktoolbox.intermediate.components.YesNoComponent
+import org.dataland.frameworktoolbox.intermediate.group.ComponentGroup
+import org.dataland.frameworktoolbox.intermediate.group.create
+import org.dataland.frameworktoolbox.intermediate.group.edit
+import org.dataland.frameworktoolbox.intermediate.group.getOrNull
+import org.dataland.frameworktoolbox.intermediate.logic.DependsOnComponentValue
+import org.dataland.frameworktoolbox.intermediate.logic.FrameworkConditional
+
+/**
+ * This object contains implementations for the rolling window components in the umwelt category of the GDV framework
```

Comment:
One question: Why do we need separate logic for different business rolling components? Couldn't those be simplified into one general logic?

Reply:
To make linter happy